tasks/task01/NarcissisticNumber.py

```diff
@@ -0,0 +1,11 @@
+for k in range(1, 1001):
+    numberToProcess = k
+    listOfNumbers = list()
+    while numberToProcess != 0:
+        listOfNumbers.append(numberToProcess % 10)
+        numberToProcess = numberToProcess // 10
+    sumOfDigits = 0
+    for i in listOfNumbers:
+        sumOfDigits += i ** len(listOfNumbers)
+    if sumOfDigits == k:
+        print("{}, n={}".format(k, len(listOfNumbers)))
```

Comment:
Все работает отлично. Только хотел сказать, что в Python 3.6 добавили более удобный способ форматировать строки - [f-strings](https://realpython.com/python-f-strings/#f-strings-a-new-and-improved-way-to-format-strings-in-python). С ними будет понятней:
```suggestion
        print(f"{k}, n={len(listOfNumbers)}")
```

Reply:
done